CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ target_include_directories(hss1394 PRIVATE
 )
 
 if(APPLE)
-  target_sources(hss1394 PRIVATE src/osx/1394API_OSX.cpp)
+  target_sources(hss1394 PRIVATE src/macos/1394API_OSX.cpp)
```

Comment:
and the filename: 1394API_MAC.cpp or 1394_MACOS.cpp or 1394_macOS.cpp. Take your pick